linode/manager
# refactor: [M3-7437] - Update Profile Queries

> [!important]
> Typecheck, Unit Tests, and E2E test will fail and Cloud Manager will not function yet
> Things will start slowly working again as we continue refactor 

## Description 📝

- Adds new package to handle typesafe query keys 🔑📦
- Refactors all profile related queries and mutations to use new format and query key ♻️

## How to test 🧪

- **Nothing is going to work yet** (things should start working after we update account queries, which I will do next)
- Check the diff and verify the changes are solid

## As an Author I have considered 🤔
